pkg/ui/workspaces/cluster-ui/src/store/localStorage/localStorage.reducer.ts

```diff
@@ -53,11 +59,12 @@ export type LocalStorageState = {
   "sortSetting/JobsPage": SortSetting;
   "sortSetting/InsightsPage": SortSetting;
   "sortSetting/SchemaInsightsPage": SortSetting;
+  "sortSetting/DatabasesPage": SortSetting;
```

Comment:
You can create keys for this just as you did above for the details page.

Reply:
Done.